src/Raven.Client/Documents/Commands/Batches/ICommandData.cs

```diff
@@ -32,16 +32,14 @@ public enum CommandType
         AttachmentCOPY,
         CompareExchangePUT,
         CompareExchangeDELETE,
-
+        ForceRevisionCreation, 
```

Comment:
Maybe just `CreateRevision` ? Thoughts?

Reply:
imho - I prefer **ForceRevisionCreation** because this is not a regular revision - 
this command is created from the special flow of forcing a revision - 
as opposed to a revision created just because the Revisions Settings are set.